aten/src/ATen/nn_parse.py

```diff
@@ -295,10 +295,10 @@ def initialize_output_arg(arg):
         arg['is_nullable'] = True
 
         # grad_weight and grad_bias need to be resized and zeroed
-        if arg['name'] == 'grad_weight' and base['name'] != '_thnn_conv2d':
+        if arg['name'] == 'grad_weight' and base['name'] != '_thnn_conv2d' and base['name'] != '_thnn_conv_depthwise2d':
```

Comment:
reupping earlier note - personally I'd rather have a list of names that's getting checked here, but not a blocker if you either like it better this way or just want to blast ahead

Reply:
yep, good suggestion.  I knew ahead of time (from checking the generated code) that there were only two cases and I was going to remove them in this stack so it didn't seem worth it.